src/cashier_frontend_new/src/modules/guard/components/ProtectedIP.svelte

```diff
@@ -0,0 +1,30 @@
+<script lang="ts">
+  import ProtectionProcessingState from "$modules/guard/components/ProtectionProcessingState.svelte";
+  import { userIPStore } from "$modules/guard/state/userIPStore.svelte";
+  import { type Snippet } from "svelte";
+
+  let {
+    children,
+  }: {
+    children: Snippet;
+  } = $props();
+
+  let shouldShow = $state(false);
+
+  $effect(() => {
+    if (userIPStore.countryCode && userIPStore.isBlacklisted()) {
+      alert(
+        "Sorry for the inconvenience, access from your location is restricted.",
```

Comment:
I propose to use translations across the app, as we do in other parts of the frontend app. I think you may not update it, but in the end I will go through the texts and will replace them with translations. Both options work I think.